book/website/community-handbook/local-build.md

```diff
@@ -39,6 +39,20 @@ You will need to locate your "terminal" or "prompt" application on your machine.
    git clone https://github.com/the-turing-way/the-turing-way
    ```
 
+   ```{note}
+   To address potential barriers with slow internet connections due to the large size of this repository,    
+
+   you can use [partial clones](https://github.blog/2020-12-21-get-up-to-speed-with-partial-clone-and-shallow-clone/#). Specifically, focusing on blobless clones, which are efficient for developers, involves utilizing the --filter=blob:none option in the git clone command.
+
+   By using --filter=blob:none, the initial git clone operation downloads all reachable commits and trees, while blobs (file contents) for commits are only downloaded when performing a `git checkout`.
+   
+   Here's the command to use Blobless clones:
+
+   
+   `git clone --filter=blob:none https://github.com/the-turing-way/the-turing-way.git`
```

Comment:
I'm not sure whether we can do anything about it, but it would be nice to force this command to be in the same line when rendered (currently there is a linebreak). Same above for `git checkout`, it renders into two different lines.

So, I would suggest to add the commands in via a console line, just as it's done with the other pip and git commands in the same page.